Framework/CurveFitting/src/FuncMinimizers/FABADAMinimizer.cpp

```diff
@@ -976,53 +650,482 @@ void FABADAMinimizer::SimAnnealingRefrigeration() {
     m_changes[j] = 0;
   }
   // Simulated Annealing variables updated
-  --m_LeftRefrPoints;
+  --m_leftRefrPoints;
   // To avoid numerical error accumulation
-  if (m_LeftRefrPoints == 0)
-    m_Temperature = 1.0;
+  if (m_leftRefrPoints == 0)
+    m_temperature = 1.0;
   else
-    m_Temperature /= m_TempStep;
+    m_temperature /= m_tempStep;
 }
 
-bool FABADAMinimizer::IterationContinuation() {
+/* Returns true if iteration must continue. Returns false otherwise.
+*
+*/
+bool FABADAMinimizer::iterationContinuation() {
 
   // If still through Simulated Annealing
-  if (m_LeftRefrPoints != 0)
+  if (m_leftRefrPoints != 0)
     return true;
 
   if (!m_converged) {
 
     // If there is not convergence continue the iterations.
-    if (m_counterGlobal < m_max_iter - m_ChainIterations) {
+    if (m_counterGlobal < m_maxIter - m_chainIterations) {
       return true;
     }
     // If there is not convergence, but it has been made
     // convergenceMaxIterations iterations, stop and throw the error.
     else {
       std::string failed = "";
       for (size_t i = 0; i < m_nParams; ++i) {
-        if (!m_par_converged[i]) {
-          failed.append(m_FitFunction->parameterName(i)).append(", ");
+        if (!m_parConverged[i]) {
+          failed = failed + m_fitFunction->parameterName(i) + ", ";
         }
       }
       failed.replace(failed.end() - 2, failed.end(), ".");
       throw std::runtime_error(
           "Convegence NOT reached after " +
-          std::to_string(m_max_iter - m_ChainIterations) +
+          std::to_string(m_maxIter - m_chainIterations) +
           " iterations.\n   Try to set better initial values for parameters: " +
           failed + " Or increase the maximum number of iterations "
                    "(MaxIterations property).");
     }
   } else {
     // If convergence has been reached, continue until we complete the chain
     // length. Otherwise, stop interations.
-    return m_counter < m_ChainIterations;
+    return m_counter < m_chainIterations;
   }
   // can we even get here? -> Nope (we should not, so we do not want it to
   // continue)
   return false;
 }
 
+/** Create the workspace for the complete parameters chain (the last histogram
+*is for the Chi square).
+*
```

Comment:
It refers to the fitting function parameters, not to the c++ function parameters. Maybe I misunderstood you, if so, I would be grateful if you clarified it to me